components/LeagueEntries/LeagueEntries.tsx

```diff
@@ -107,14 +143,22 @@ const LeagueEntries = ({
           data-testid="league-entry-pick-button-container"
         >
           {!isEliminated && (
-            <Link href={linkUrl} data-testid="league-entry-pick-button-link">
-              <Button
-                className="league-entry-pick-button"
-                data-testid="league-entry-pick-button"
-                label={isPickSet ? 'Change Pick' : 'Make Pick'}
-                variant={isPickSet ? 'secondary' : 'default'}
-              />
-            </Link>
+            <LinkCustom
```

Comment:
There are 3 conditions in this component. This is best, it's having conditional statements for 2 separate components. It's easier to read and maintain.